dependencies.list

```diff
@@ -1,3 +1,3 @@
 VERSION=10.51.0
-REALM_CORE_VERSION=v14.9.0
+REALM_CORE_VERSION=v14.10.0-8-g8d7134760
```

Comment:
Core 14.10.0 was released, we probably want to point to that

Reply:
`v14.10.0-8-g8d7134760` is newer than v14.10.0. git describe's format is `${tagName}-${numberOfCommitsSinceTag}-`${commitHash}`, so this is referring to commit g8d7134760, which is 8 commits after tag v14.10.0.